libs/ui-toolkit/src/components/dialog/dialog.tsx

```diff
@@ -59,7 +61,10 @@ export function Dialog({
           className="fixed inset-0 bg-black/50 z-10"
           data-testid="dialog-overlay"
         />
-        <DialogPrimitives.Content className={contentClasses}>
+        <DialogPrimitives.Content
+          className={contentClasses}
+          onCloseAutoFocus={(e) => onCloseAutoFocus?.(e)}
```

Comment:
chore (non blocking): passing undefined here should be fine

```suggestion
          onCloseAutoFocus={onCloseAutoFocus}
```